### DISCUSSION TITLE
Example useage of chord style shortcuts ("easymotion") useage example?

### DISCUSSION THREAD

**Discussion by MikeLemo1** [junegunn/fzf#3659] [Q&A]
Is it possible to get a simple easy explation of how to cleanly use the `jump` from `man fzf` which I assume is a way to bind key
chords in fzf menues, I'm looking for somethign say pressing ++jk++ (++j-k++ OR j `chord` k) to toggle hide/display of fzf preview?

**Comment by LangLangBart** [comment]
> simple easy explanation of how to cleanly use the `jump` from `man fzf`

`jump` allows you to navigate to a specific line without the need for arrow keys or scrolling. For
instance:

Type the command below and press either <kbd>;</kbd> or <kbd>:</kbd>. Afterward, press the symbol
corresponding to the item you wish to `jump` to.

```bash
fzf  --bind ';:jump' --bind '::jump-accept'
```

See some GIFs in a related discussion: #3281

> which I assume is a way to bind key chords in fzf menues, I'm looking for somethign say pressing
> ++jk++ (++j-k++ OR j chord k) to toggle hide/display of fzf preview?

I don't think such a feature currently exists in `fzf`, all possible keybindings are listed in the `man fzf`

```bash
man fzf | less --pattern 'AVAILABLE KEYS'
```
